### PR TITLE
Updates in alignment model

### PR DESCRIPTION
**Pull request by czapplitec**
# Update

- IfcSine: 
  - added ConstantTerm, link to #56 
- IfcThirdOrderPolynomialSpiral: 
  - QubicTerm to CubicTerm
- IfcAlignmentVerticalSegment: 
  - HorizontalLength, type changed to IfcNonNegativeLengthMeasure, link to #53 
  - RadiusOfCurvature, type changed to IfcLengthMeasure, link to #54 
- IfcAlignmentCantSegment: 
  - HorizontalLength, type changed to IfcNonNegativeLengthMeasure, link to #53

### CODE REVIEW

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Entities/IfcAlignmentVerticalSegment/DocEntity.xml [hunk 1]
```diff
@@ -22,8 +22,8 @@
 		<DocAttribute Name="EndGradient" UniqueId="f91fe698-cbc3-48f4-80f1-5b8a5b0c6f49" DefinedType="IfcRatioMeasure">
 			<Documentation>End gradient of the segment. In the case of a PredefinedType=&apos;.CONSTANTGRADIENT.&apos; the value is the same as _StartGradient_.</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="RadiusOfCurvature" UniqueId="4c28f1e4-ef03-421a-a559-862cc0c31b7f" DefinedType="IfcPositiveLengthMeasure" AttributeFlags="1">
-			<Documentation>Radius of parabola or arc.
+		<DocAttribute Name="RadiusOfCurvature" UniqueId="4c28f1e4-ef03-421a-a559-862cc0c31b7f" DefinedType="IfcLengthMeasure" AttributeFlags="1">
+			<Documentation>Radius of parabola or arc. Positive values imply a CCW direction whereas negative CW, looked from the right side of alignment.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Entities/IfcAlignmentVerticalSegment/DocEntity.xml:26]: What is the right side of alignment?

**Review by SergejMuhic — COMMENTED** [review]
Was wondering about the right side of the alignment.

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Entities/IfcAlignmentVerticalSegment/DocEntity.xml (hunk 1 above)
Comment (czapplitec) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Entities/IfcAlignmentVerticalSegment/DocEntity.xml]: Right side just means from right-hand side when looking into the direction of alignment, but maybe confusing and not necessary to say this. I have removed it.

**Review by SergejMuhic — APPROVED** [review]
Looks good. Merging.

**Pull request merged**